package.json

```diff
@@ -29,7 +29,7 @@
     "@babel/core": "^7.14.6",
     "@babel/preset-env": "^7.14.7",
     "@babel/preset-react": "^7.14.5",
-    "@hypothesis/frontend-shared": "3.3.0",
+    "@hypothesis/frontend-shared": "3.4.0",
```

Comment:
I merged a PR that updates the frontend-shared package to v3.4.0, so if you rebase on master that will disappear from this PR.